osu.Server.Spectator.Tests/Multiplayer/MultiplayerCountdownTest.cs

```diff
@@ -9,14 +9,13 @@
 using Moq;
 using osu.Game.Online.Multiplayer;
 using osu.Game.Online.Multiplayer.Countdown;
-using osu.Server.Spectator.Hubs;
 using Xunit;
 
 namespace osu.Server.Spectator.Tests.Multiplayer
 {
     public class MultiplayerCountdownTest : MultiplayerTest
     {
-        [Fact]
+        [Fact(Timeout = 1000)]
```

Comment:
Not sure about these timeouts... I guess we can keep them until they cause intermittent CI failures, seem unnecessary otherwise.

Reply:
Speak of the devil (CI failure) :smile:

Reply:
The idea was to avoid runaway tests that never end (if we happen to break something). Should I set them to some higher number like 60s?

Reply:
Sure, I think that's reasonable.